docs/mobile_sdk.md

```diff
@@ -25,17 +25,28 @@ target 'MyApp WatchKit Extension' do
 end
 ```
 
+From the command line, type "pod install". This will install the SDK for an iOS app with a watchOS 2 extension.  Use the workspace file Cocoapods creates for your app to open the project in Xcode.
+
+### Installing using Carthage
+
+Create a file in your app's project directory all it 'Cartfile'. Put the following lines in the Cartfile:
+```
+github "openwhisk//openwhisk-client-swift.git" ~> 0.1.0 # Or latest version
+```
+
+From the command line, type 'carthage update --platform ios'. Carthage downloads and builds the SDK, creates a directory called Carthage in your app's project directory, and puts an OpenWhisk.framework file inside Carthage/build/iOS. Add OpenWhisk.framework to the embedded frameworks in your Xcode project.
+
 ### Installing from source code
 
-Source code is available at https://github.com/openwhisk/openwhisk. The SDK is in mobile/iOS/SDK.
+Source code is available at https://github.com/openwhisk//openwhisk-client-swift.git. Open project using the OpenWhisk.xcodeproj file in Xcode.  The project contains two schemes "OpenWhisk" and "OpenWhiskWatch" targetted for iOS and WathOS2, respectively.  Build the project for the targets you need and add the resulting frameworks to your app (usually in ~/Library/Developer/Xcode/DerivedData/your app name).
```

Comment:
"WathOS2" should be "watchOS 2" (see [Apple's own use](https://developer.apple.com/watchos/)).